src/test/regress/sql/multi_subquery_behavioral_analytics.sql

```diff
@@ -1285,6 +1285,59 @@ SELECT * FROM run_command_on_workers('DROP FUNCTION array_index(ANYARRAY, ANYELE
 ORDER BY 1,2;
 DROP FUNCTION array_index(ANYARRAY, ANYELEMENT);
 
+-- a not supported query due to constant range table entry
+SELECT count(*) as subquery_count
+FROM (
+  SELECT 
+      user_id
+    FROM
+    users_table
+  WHERE
+    (value_1 = '5' OR value_1 = '13')
+  GROUP BY user_id                                                                          
+  HAVING count(distinct value_1) = 2
+  ) as a
+  LEFT JOIN (
+  SELECT
+    1 as user_id
+  ) AS b 
+  ON a.user_id = b.user_id 
+WHERE b.user_id IS NULL
+GROUP BY a.user_id;
+
+-- same with INNER JOIN
+SELECT count(*) as subquery_count
+FROM (
+  SELECT 
+      user_id
+    FROM
+    users_table
+  WHERE
+    (value_1 = '5' OR value_1 = '13')
+  GROUP BY user_id                                                                          
+  HAVING count(distinct value_1) = 2
+  ) as a
+  INNER JOIN (
+  SELECT
+    1 as user_id
+  ) AS b 
+  ON a.user_id = b.user_id 
+WHERE b.user_id IS NULL
+GROUP BY a.user_id;
+
+-- this is slightly different, we use RTE_VALUEs here
+SELECT Count(*) AS subquery_count 
+FROM   (SELECT user_id 
+        FROM   users_table 
+        WHERE  ( value_1 = '5' 
+                  OR value_1 = '13' ) 
```

Comment:
Looking to pattern here ` OR value_1 = '13' ) ` should continue on the line above.